tools/mo/requirements.txt

```diff
@@ -5,7 +5,7 @@ mxnet~=1.7.0.post2; sys_platform != 'win32'
 networkx~=2.5; python_version <= "3.6"
 networkx<2.8.1; python_version > "3.6"
 protobuf>=3.18.1,<4.0.0
-onnx>=1.8.1,<1.12
+onnx==1.12.0
```

Comment:
Does MO should use only one specific version? I do not know all of the internals, but supporting all from `1.8.1` to (inlcuding) `1.12.0` should be doable as MO is "semi-standalone" project. Please verify that.